### PR TITLE
Add Operational Risk Violations data to scan and build-scan commands

### PR DESCRIPTION
**Pull request by barbelity**
Add operational risk violations table to build-scan and scan commands.

Basic:
image

Extended:
image

Extended in a narrow terminal:
image

- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

### CODE REVIEW

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
@barbelity -
The new "operational risks" tables includes a lot of columns. Does it also look well with a very narrow terminal?
See for example how the output table of the "jf docker scan" gets shriked nicely on a narrow terminal - 
image
I'm just making sure we're not introducing here a UX issue.

File: xray/utils/resultstable.go [hunk 1]
```diff
@@ -76,9 +87,9 @@ func PrepareViolationsTable(violations []services.Violation, multipleRoots, colo
 					},
 				)
 			}
-		} else {
-			// License compliance violation
+		case "license":
 			for compIndex := 0; compIndex < len(impactedPackagesNames); compIndex++ {
+
```
[the remark this answers is not in the document]
Comment (barbelity) [reply xray/utils/resultstable.go:92]: Removed.

File: xray/utils/resultstable.go [hunk 2]
```diff
@@ -427,3 +474,38 @@ func getSeverity(severityTitle string) *severity {
...
+	row.cadence = cadence
+	row.commits = commits
+	row.committers = committers
+	row.newerVersions = newerVersions
+	row.latestVersion = latestVersion
+	row.riskReason = violation.RiskReason
+	row.eolMessage = violation.EolMessage
+}
+
+func getOperationalRiskReadableData(violation services.Violation) (isEol, cadence, commits, committers, newerVersions, latestVersion string) {
```
[the remark this answers is not in the document]
Comment (barbelity) [reply xray/utils/resultstable.go:490]: Edited this function to return a struct containing the operational-risk data.
Also changed the operational-risk case to call this function only once, as the fetched data from the violation is the same for all rows constructed in the loop.

File: xray/utils/resultstable.go [hunk 3]
```diff
@@ -91,6 +102,21 @@ func PrepareViolationsTable(violations []services.Violation, multipleRoots, colo
...
+					impactedPackageName:    impactedPackagesNames[compIndex],
+					impactedPackageVersion: impactedPackagesVersions[compIndex],
+					impactedPackageType:    impactedPackagesTypes[compIndex],
+					component:              components[compIndex],
+				}
+				populateOperationalRiskRowData(operationalRiskViolationsRow, violation)
+				operationalRiskViolationsRows = append(operationalRiskViolationsRows, *operationalRiskViolationsRow)
+			}
+		default:
+			// Unsupported type, ignore
```
[the remark this answers is not in the document]
Comment (barbelity) [reply xray/utils/resultstable.go:119]: Added this empty 'default' due to what we experienced after adding the Operational-Risk violations to the scan result, as the operational risk violations were printed in the Licenses-violations table.
I didn't add an error there for a future case of a new violation-type added in Xray, so it won't break the command.

What do you think?

[the remark this answers is not in the document]
Comment (eyalbe4) [reply xray/utils/resultstable.go:126]: Got it. OK.

**Pull request merged**